nordicsemi/version.py

```diff
@@ -37,4 +37,4 @@
 
 """ Version definition for nrfutil. """
 
-NRFUTIL_VERSION = "2.2.0"
+NRFUTIL_VERSION = "2.3.0"
```

Comment:
2.3.0 has now been released, so this should be bumped to 2.4.0. There is also a merge conflict that needs to be resolved.